From b47cec9f447825080a75345b57a9be5b19fff575 Mon Sep 17 00:00:00 2001
From: Vadim Pasternak <vadimp@nvidia.com>
Date: Sun, 3 Jul 2022 10:33:46 +0300
Subject: [PATCH i2c backport v4.19 v2 1/1] mlxsw: i2c: Increase transactions
 timeout window

The time out window is increased due to the latest changes in firmware
causing increasing the processing time of some transactions.
There are few type of transaction, which could take very long time
causing time out of other transaction, waiting in mailbox queue.
Thus, current window (5 sec) is not sufficient and doubled in order to
get transaction completed.

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>
---
 drivers/net/ethernet/mellanox/mlxsw/i2c.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/mellanox/mlxsw/i2c.c b/drivers/net/ethernet/mellanox/mlxsw/i2c.c
index 3b6f73e4a..f65d9eebe 100644
--- a/drivers/net/ethernet/mellanox/mlxsw/i2c.c
+++ b/drivers/net/ethernet/mellanox/mlxsw/i2c.c
@@ -49,7 +49,7 @@
 #define MLXSW_I2C_ADDR_BUF_SIZE		4
 #define MLXSW_I2C_BLK_DEF		32
 #define MLXSW_I2C_RETRY			5
-#define MLXSW_I2C_TIMEOUT_MSECS		5000
+#define MLXSW_I2C_TIMEOUT_MSECS		10000
 #define MLXSW_I2C_MAX_DATA_SIZE		256
 
 #define MLXSW_I2C_WORK_ARMED		1
-- 
2.20.1

